### DISCUSSION TITLE
No Request on UI

### DISCUSSION THREAD

**Discussion by m7mdcc** [Dokploy/dokploy#498] [Q&A] [answered]
![Screenshot 2024-09-29 194522](https://github.com/user-attachments/assets/052a56d9-2061-4f35-8e6a-098ad48ae3d5)
Hi team , why i'm not seeing any requests here ? 
I'm using canary version

**Comment by Siumauricio** [comment]
Can you try to reload traefik?

**Comment by m7mdcc** [comment]
I have tried that many times , restarting traefik and server itself

**Comment by m7mdcc** [comment, marked as answer]
After update to v0.9.4 , issue solved
